.github/workflows/daml-all-in-one.yaml

```diff
@@ -0,0 +1,56 @@
+name: daml-all-in-one-publish
+
+on:
+  # Publish `v1.2.3` tags as releases.
+  push:
+    tags:
+      - v*
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+env:
+  IMAGE_NAME: cactus-daml-all-in-one
```

Comment:
@raynatopedrajeta Sorry for not catching this earlier: please call the image `cacti-daml-all-in-one` instead.